create-labels.py

```diff
@@ -0,0 +1,100 @@
+# This script removes default github labels and adds custom ones using python requests and the github API
+# To work it needs: 1) User token, 2) Target username / organization name and target repository name and the 
+# By Pablo and Spearbit with <3
+# License: MIT
+
+import requests
+
+
+def delete_default_labels():
+    default_labels = [
+        "bug", 
+        "duplicate",
+        "enhancement", 
+        "invalid", 
+        "question", 
+        "wontfix", 
+        "question", 
+        "good first issue", 
+        "help wanted", 
+        "documentation"
+        ]
+    for i in default_labels:
+        r = requests.delete(f"https://api.github.com/repos/{username}/{repo}/labels/{i}", headers=headers)
+        print(r)
+    print("Done deleting default labels!\n")
+
+
+def create_new_labels():
+    new_labels = [
+        "Severity: Critical Risk", 
+        "Severity: High Risk",
+        "Severity: Medium Risk", 
+        "Severity: Low Risk", 
+        "Severity: Informational",
+        "Severity: Gas Optimization",
+        "Status: Acknowledged",
+        "Status: Fixed",
+        "Status: ReadyForReport"
+    ]
+
+    colors = [
+        "ff0000",
+        "B60205",
+        "D93F0B",
+        "FBCA04",
+        "1D76DB",
+        "B4E197",
+        "5319E7",
+        "0E8A16",
+        "bfdadc"
+    ]
+    i = 0
+    if len(new_labels) == len(colors):
+        for n in new_labels:
+            data={
+                "name":n,
+                "color":colors[i]
+            }
+            print(data)
+            r = requests.post(f"https://api.github.com/repos/{username}/{repo}/labels", headers=headers, json=data)

```

Comment:
Any reason you went with a raw request rather than using pygithub?

Reply:
Did not think that using a wrapper was necessary for such trivial task, it makes the script lighter and easier to modify for anyone